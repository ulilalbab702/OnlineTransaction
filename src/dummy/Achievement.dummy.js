export const Achievement = [
    {
        "Code": "FM-JBYPM",
        "Name": "FMC Jembayan PAMA",
        "Percent": 50,
        "BackLogOpen": 1,
        "BackLogClose": 1,
        "Priority1": 1,
        "Priority2": 0,
        "Priority3": 1,
        "Priority4": 0
    }
]