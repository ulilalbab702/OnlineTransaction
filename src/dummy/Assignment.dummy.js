export const listOfMechanicDummy = [
    {
      "Username": "string",
      "FirstName": "string",
      "LastName": "string",
      "Email": "string",
      "MechanicUT": true,
      "Jobs": [
        {
          "JobType": "string",
          "Total": 0
        }
      ]
    }
  ]
export const jobSummaryDummy = {
  "Date": "2020-06-04T16:18:05.5472055",
  "CreatedBy": "AGUS LISANTO",
  "AssignedBy": "AGUS LISANTO",
  "MechanicLeader": {
      "UserName": "82107038@unitedtractors.com",
      "Name": "SUPRIYADI SUPRIYADI"
  },
  "Mechanics": [],
  "HandoverDate": null,
  "HandoverCreatedBy": null,
  "HandoverAssignedBy": null,
  "HandoverMechanicLeader": null,
  "HandoverMechanics": null,
  "Job": {
      "WoId": "8ff577d1-2911-4bbd-9d55-ffad4fe76fc5",
      "WoNumber": "71018650",
      "JobType": "INS",
      "UnitModel": "HD465-7",
      "SerialNumber": "15163",
      "UnitCode": "HT3022",
      "EquipmentNumber": "71506",
      "IsHandover": false,
      "Id": "a162bd06-6aca-499a-9f97-a5839b8a0612",
      "CreatedBy": "80107002@unitedtractors.com",
      "CreatedDate": "2020-06-04T16:18:05.5472055",
      "UpdatedBy": "80107002@unitedtractors.com",
      "UpdatedDate": "2020-06-04T16:18:05.5472055",
      "IsActive": true
  }
}