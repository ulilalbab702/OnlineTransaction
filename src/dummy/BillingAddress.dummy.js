export const dataBillingAddressDummy = [
    {
        "fullName": "Test Integrasi",
        "contactNumber": "088122345",
        "email": "testintegrasi@unitedtractors.com",
        "position": "string",
        "addressLabel": "Kantor PT Global Service Indonesia",
        "city": "Jakarta",
        "address": "Jl Raya kaliabang Tengah, Ruko San Jaya, Blok A5 No 9,\r\nSebalah Indomaret",
        "postalCode": "13920",
        "locationLatitude": "3.669276961596295",
        "locationLongitude": "102.4003031491451",
        "isPrimary": true
    }]