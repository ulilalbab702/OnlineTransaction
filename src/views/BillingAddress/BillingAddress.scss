body {
  background-color: #ffffff;
}
.styleCIBold {
  font-family: "SFProText-Bold";
  font-size: 22px;
  line-height: 26px;
  color: #000000;
  font-weight: bold;
  margin-bottom: 1.5;
  margin-top: 2rem;
}
.styleBABold {
  font-family: "SFUIText-Regular";
  font-size: 22px;
  line-height: 26px;
  color: #000000;
  font-weight: bold;
  margin-bottom: 1.5;
  margin-top: 2rem;
}
.formAddress {
  margin-left: 0;
  padding: 0;
}

label {
  font-family: "Lato-Regular";
  font-size: 12px;
  line-height: 14px;
  color: #707070;
  font-weight: normal;
}

.styleIsiForm {
  font-family: "SFUIText-Regular";
  font-size: 14px;
  line-height: 17px;
  color: #000000;
  font-weight: normal;
  font-style: normal;
  border: 1px solid rgba(0, 0, 0, 0.5);
  height: 41px;
  border-radius: 8px;
  &::placeholder {
    font-family: "SFUIText-Regular";
    font-size: 14px;
    line-height: 17px;
    color: #000000;
    font-weight: normal;
    font-style: normal;
  }
}
.styleIsiForm:disabled {
  background-color: #ffffff;
}
.titleBtn {
  font-size: 14px;
  font-family: "SFProText-Bold";
  line-height: 17px;
  font-style: normal;
  font-weight: 600;
  text-align: center;
  margin-bottom: 0;
  color: rgba(0, 0, 0, 0.5);
  float: right;
}
.btnContinue {
  align-items: center;
  display: flex;
  justify-content: center;
  background-color: #ffd500;
  border-radius: 8px;
  width: 331px;
  height: 48px;
  float: right;
  cursor: pointer;
}
.conModalPinPoint {
  position: absolute;
  top: 50%;
  left: 50%;
  height: 600px;
  width: 500px;
  padding: 20px !important;
  background-color: #ffffff;
  transform: translate(-50%, -50%);
  border-radius: 10px;
}
.modalPinTitle {
  font-family: "SFProText-Bold";
  font-style: normal;
  font-weight: 600;
  font-size: 20px;
  line-height: 24px;
  text-align: center;
  color: #000000;
  margin: 0;
  padding: 0;
  margin-bottom: 1rem;
}
.cardPurchaseSummary {
  margin-top: 2rem;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  box-shadow: 0 0px 0px 0 rgba(0, 0, 0, 0), 2px 5px 15px 0 rgba(0, 0, 0, 0.1);
  border-radius: 20px;
  background: #ffffff;
  padding: 0;
  margin-bottom: 1.5rem;
  width: 400px;
  height: auto;
  border-radius: 16px;
}
.kontenPurchaseSummary {
  padding: 1rem 1.5rem;
  padding-bottom: 0;
}
.teksCardPurchaseSummary {
  font-family: "SFUIText-Regular";
  font-style: normal;
  font-weight: bold;
  font-size: 16px;
  line-height: 19px;
  color: #000000;
}
.subTotalPS {
  display: flex;
  justify-content: space-between;
  margin-right: 0.5px;
  margin-left: 1px;
}
.teksSubTotalPS {
  align-items: flex-start;
  font-family: "NunitoSans-Regular";
  font-style: normal;
  font-weight: 400;
  font-size: 12px;
  line-height: 15px;
  color: #404040;
  margin-top: 1rem;
}
.hargaSubTotalPS {
  align-items: flex-end;
  font-family: "NunitoSans-Regular";
  font-style: normal;
  font-weight: 600;
  font-size: 13px;
  line-height: 16px;
  color: #676767;
  margin-top: 1rem;
}
.hargaSubTotal1PS {
  align-items: flex-end;
  font-family: "NunitoSans-Bold";
  font-style: normal;
  font-weight: 800;
  font-size: 14px;
  line-height: 19px;
  color: #fa591d;
  margin-top: 1rem;
}
.rowSelectAddress {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  margin-top: 2vw;
  padding: 1vw 1.5vw;
  border-radius: 1.2vw;
  background-color: #ffffff;
  box-shadow: 0 0px 0px 0 rgba(0, 0, 0, 0), 2px 5px 15px 0 rgba(0, 0, 0, 0.1);
}
.divIconDelivery {
  display: flex;
  flex-grow: 0.5;
  align-items: center;
}
.iconDelivery {
  width: 3.2vw;
}
.divTextDelivery {
  display: flex;
  flex-grow: 4;
  align-items: center;
}
.textDelivery {
  font-family: "SFProText-Bold";
  font-size: 1.2vw;
  line-height: 2vw;
  margin: 0;
}
.divBtnSelectAddress {
  display: flex;
  align-items: center;
  justify-content: center;
  width: 14vw;
  height: 3.2vw;
  background: #ffd500;
  border-radius: 0.5vw;
  cursor: pointer;
}
.textBtnSelectAddress {
  font-family: "SFUIDisplay-Regular";
  font-style: normal;
  font-size: 1vw;
  line-height: 1.5vw;
  margin: 0;
}
.boxDeliveryAddress {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  margin-top: 2vw;
  padding: 1vw 1.5vw;
  border-radius: 1.2vw;
  background-color: #f9f9f9;
}
.labelDeliveryAddress {
  font-family: "SFProText-Bold";
  font-style: normal;
  font-weight: bold;
  font-size: 1.5vw;
  line-height: 21px;
  color: #050505;
}
.nameDeliveryAddress {
  font-family: "SFProText-Regular";
  font-style: normal;
  font-size: 1.2vw;
  line-height: 2vw;
  letter-spacing: 0.01em;
  color: #000000;
  opacity: 0.4;
  margin-bottom: 0px;
}
.detailDeliveryAddress {
  font-family: "SFProText-Regular";
  font-style: normal;
  font-size: 1.2vw;
  line-height: 2vw;
  letter-spacing: 0.01em;
  color: #000000;
  opacity: 0.4;
}
.actionDeliveryAddress {
  font-family: "SFProText-Bold";
  font-style: normal;
  font-size: 1.25vw;
  line-height: 1.4vw;
  color: #148f0f;
  opacity: 0.75;
  margin: 0;
  cursor: pointer;
}

.modal-confirm-ktp {
  display: flex;
  justify-content: center;
  align-items: center;
  justify-items: center;
  width: 40%;
  height: fit-content;
  margin: auto;
  padding: 0;
  .container-modal-shippment {
    display: flex;
    justify-content: center;
    align-items: center;
    padding: 0;
    .confirm-modal-body {
      background-color: white;
      margin: auto;
      border-radius: vw(15);
      display: flex;
      flex-direction: column;
      align-content: center;
      padding: 2vw 3vw;
      width: 100%;
      .titleConfirm {
        font-family: "SFProText-Bold";
        font-size: vw(24);
        color: #000000;
        text-align: center;
      }
      .iconModal {
        width: 35%;
      }
      .titleAvailable {
        font-family: "SFUIDisplay-Regular";
        font-style: normal;
        font-weight: normal;
        font-size: 18px;
        line-height: 26px;
        text-align: center;
        letter-spacing: 0.3px;
        color: #636363;
      }
      .btn {
        cursor: pointer;
        height: 50px;
        width: 40%;
        border-radius: 8px;
        display: flex;
        justify-content: center;
        align-items: center;
        margin: 0 2%;
        p {
          font-family: "SFProText-Bold";
          font-style: normal;
          font-weight: bold;
          font-size: 14px;
          line-height: 17px;
          color: #000000;
          margin-bottom: 0;
        }
      }
      .con-more-info-ktp {
        width: 100%;
        height: auto;
        margin-top: 10px;
        display: flex;
        flex-direction: column;
        justify-content: center;
        align-items: center;
        span {
          font-family: "SFUIDisplay-Regular";
          font-style: normal;
          font-weight: normal;
          font-size: 14px;
          line-height: 14px;
          color: #000000;
          margin-top: 20px;
          margin-bottom: 14px;
        }
        .con-more-info-image {
          width: 50%;
          display: flex;
          justify-content: space-evenly;
          .marginIcon {
            margin-top: 4px;
          }
        }
      }
    }
  }
}

@media only screen and (min-width: 1280px) {
  .paddingBreadCrumps {
    padding-left: 10vw;
    padding-top: 7vw;
  }
  .paddingContactInformation {
    display: flex;
    flex-direction: row;
    padding-top: 0;
    margin-bottom: 5rem;
    margin-right: 0;
  }
  .contactInformation {
    padding: 0;
    padding-left: 10vw;
    padding-right: 6vw;
  }
  .styleCIBold {
    font-family: "SFProText-Bold";
    font-size: 22px;
    line-height: 26px;
    color: #000000;
    font-weight: bold;
    margin-bottom: 1.5;
    margin-top: 2rem;
  }
  .styleBABold {
    font-family: "SFUIText-Regular";
    font-size: 22px;
    line-height: 26px;
    color: #000000;
    font-weight: bold;
    margin-bottom: 1.5;
    margin-top: 2rem;
  }
  .containerPurchaseSummary {
    padding-right: 14vw;
  }
}

@media only screen and (min-width: 992px) and (max-width: 1279px) {
  .paddingBreadCrumps {
    padding-left: 3rem;
    padding-top: 7vw;
  }
  .paddingContactInformation {
    display: flex;
    flex-direction: row;
    padding-top: 0;
    margin-bottom: 5rem;
    margin-right: 0;
  }
  .contactInformation {
    padding: 0;
    padding-left: 3rem;
    padding-right: 7vw;
  }
  .styleCIBold {
    font-family: "SFProText-Bold";
    font-size: 22px;
    line-height: 26px;
    color: #000000;
    font-weight: bold;
    margin-bottom: 1.5;
    margin-top: 2rem;
  }
  .styleBABold {
    font-family: "SFUIText-Regular";
    font-size: 22px;
    line-height: 26px;
    color: #000000;
    font-weight: bold;
    margin-bottom: 1.5;
    margin-top: 2rem;
  }
  .formAddress {
    margin-left: 0;
    padding: 0;
  }

  .titleBtn {
    font-size: 12px;
    font-family: "SFProText-Bold";
    line-height: 15px;
    font-style: normal;
    font-weight: 600;
    text-align: center;
    margin-bottom: 0;
    color: rgba(0, 0, 0, 0.5);
    float: right;
  }
  .btnContinue {
    align-items: center;
    display: flex;
    justify-content: center;
    background-color: #ffd500;
    border-radius: 8px;
    width: 25vw;
    height: 48px;
    float: right;
    cursor: pointer;
  }
  .containerPurchaseSummary {
    padding-right: 3vw;
  }
}

//mobile
@media only screen and (min-width: 480px) and (max-width: 767px) {
  .paddingBreadCrumps {
    padding-left: 4vw;
  }
  .paddingContactInformation {
    display: flex;
    flex-direction: column;
    align-items: center;
    padding-top: 0;
  }
  .styleCIBold {
    padding: 0;
    margin-bottom: 1.5;
    margin-top: 2rem;
    font-size: 19px;
    line-height: 24px;
  }
  .styleBABold {
    padding-left: 0;
    margin-bottom: 1.5;
    margin-top: 2rem;
    font-size: 19px;
    line-height: 24px;
  }
  .contactInformation {
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    padding: 0 4vw;
  }
  .formAddress {
    margin: 0;
    padding: 0;
  }
  .titleBtn {
    font-size: 14px;
    font-family: "SFProText-Bold";
    line-height: 17px;
    font-style: normal;
    font-weight: 600;
    text-align: center;
    margin-bottom: 0;
    color: rgba(0, 0, 0, 0.5);
    float: right;
  }
  .cardPurchaseSummary {
    width: 450px;
  }
}

@media only screen and (min-width: 360px) and (max-width: 479px) {
  .paddingBreadCrumps {
    padding-left: 4vw;
  }
  .paddingContactInformation {
    display: flex;
    flex-direction: column;
    align-items: center;
    padding-top: 0;
  }
  .styleCIBold {
    padding: 0;
    margin-bottom: 1.5;
    margin-top: 2rem;
    font-size: 17px;
    line-height: 20px;
  }
  .styleBABold {
    padding-left: 0;
    margin-bottom: 1.5;
    margin-top: 2rem;
    font-size: 17px;
    line-height: 20px;
  }
  .contactInformation {
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    padding: 0 4vw;
  }
  .formAddress {
    margin: 0;
    padding: 0;
  }
  .containerPurchaseSummary {
    width: 100%;
    padding: 0 4vw;
    margin-left: 0rem;
  }
  .cardPurchaseSummary {
    width: 100%;
  }
  input[type="checkbox"] {
    box-sizing: border-box;
    padding: 0;
    margin-top: 0;
  }
  .titleBtn {
    font-size: 12px;
    font-family: "SFProText-Bold";
    line-height: 15px;
    font-style: normal;
    font-weight: 600;
    text-align: center;
    margin-bottom: 0;
    color: rgba(0, 0, 0, 0.5);
    float: right;
  }
  .btnContinue {
    align-items: center;
    display: flex;
    justify-content: center;
    background-color: #ffd500;
    border-radius: 8px;
    height: 48px;
    width: 100%;
    float: right;
    cursor: pointer;
  }
}

@media only screen and (min-width: 320px) and (max-width: 359px) {
  .paddingBreadCrumps {
    padding-left: 4vw;
  }
  .paddingContactInformation {
    display: flex;
    flex-direction: column;
    align-items: center;
    padding-top: 0;
  }
  .styleCIBold {
    padding: 0;
    margin-bottom: 1.5;
    margin-top: 2rem;
    font-size: 17px;
    line-height: 20px;
  }
  .styleBABold {
    padding-left: 0;
    margin-bottom: 1.5;
    margin-top: 2rem;
    font-size: 17px;
    line-height: 20px;
  }
  .contactInformation {
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    padding: 0 4vw;
  }
  .formAddress {
    margin: 0;
    padding: 0;
  }
  .containerPurchaseSummary {
    width: 100%;
    padding: 0 4vw;
    margin-left: 0rem;
  }
  .cardPurchaseSummary {
    width: 100%;
  }
  input[type="checkbox"] {
    box-sizing: border-box;
    padding: 0;
    margin-top: 0;
  }
  .titleBtn {
    font-size: 12px;
    font-family: "SFProText-Bold";
    line-height: 15px;
    font-style: normal;
    font-weight: 600;
    text-align: center;
    margin-bottom: 0;
    color: rgba(0, 0, 0, 0.5);
    float: right;
  }
  .btnContinue {
    align-items: center;
    display: flex;
    justify-content: center;
    background-color: #ffd500;
    border-radius: 8px;
    height: 48px;
    width: 100%;
    float: right;
    cursor: pointer;
  }
}

@media only screen and (max-width: 319px) {
  .paddingBreadCrumps {
    padding-left: 4vw;
  }
  .paddingContactInformation {
    display: flex;
    flex-direction: column;
    align-items: center;
    padding-top: 0;
  }
  .styleCIBold {
    padding: 0;
    margin-bottom: 1.5;
    margin-top: 2rem;
    font-size: 17px;
    line-height: 20px;
  }
  .styleBABold {
    padding-left: 0;
    margin-bottom: 1.5;
    margin-top: 2rem;
    font-size: 17px;
    line-height: 20px;
  }
  .contactInformation {
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    padding: 0 4vw;
  }
  .formAddress {
    margin: 0;
    padding: 0;
  }
  .containerPurchaseSummary {
    width: 100%;
    padding: 0 4vw;
    margin-left: 0rem;
  }
  .cardPurchaseSummary {
    width: 100%;
  }
  input[type="checkbox"] {
    box-sizing: border-box;
    padding: 0;
    margin-top: 0;
  }
  .titleBtn {
    font-size: 12px;
    font-family: "SFProText-Bold";
    line-height: 15px;
    font-style: normal;
    font-weight: 600;
    text-align: center;
    margin-bottom: 0;
    color: rgba(0, 0, 0, 0.5);
    float: right;
  }
  .btnContinue {
    align-items: center;
    display: flex;
    justify-content: center;
    background-color: #ffd500;
    border-radius: 8px;
    height: 48px;
    width: 100%;
    float: right;
    cursor: pointer;
  }
}

//tablet
@media only screen and (min-width: 768px) and (max-width: 991px) {
  .paddingBreadCrumps {
    padding-left: 2.3rem;
  }
  .paddingContactInformation {
    display: flex;
    flex-direction: row;
    padding-top: 0;
    margin-bottom: 5rem;
    margin-right: 0;
  }
  .styleCIBold {
    font-family: "SFUIText-Regular";
    font-size: 20px;
    line-height: 24px;
    color: #000000;
    font-weight: bold;
    margin-bottom: 1.5;
    margin-top: 2rem;
    padding-left: 2rem;
  }
  .styleBABold {
    font-family: "SFUIText-Regular";
    font-size: 20px;
    line-height: 24px;
    color: #000000;
    font-weight: bold;
    margin-bottom: 1.5;
    margin-top: 2rem;
  }
  .contactInformation {
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    padding: 0;
    padding-right: 7vw;
  }
  .containerPurchaseSummary {
    margin-left: -2vw;
    padding-right: 3vw;
  }

  .titleBtn {
    font-size: 14px;
    font-family: "SFProText-Bold";
    line-height: 17px;
    font-style: normal;
    font-weight: 600;
    text-align: center;
    margin-bottom: 0;
    color: rgba(0, 0, 0, 0.5);
    float: right;
  }
  .btnContinue {
    width: 40vw;
  }
  .cardPurchaseSummary {
    width: 40vw;
  }
  .kontenPurchaseSummary {
    padding: 1rem 1rem;
  }
  .boxTotalPS {
    margin-bottom: 0 !important;
  }
}
