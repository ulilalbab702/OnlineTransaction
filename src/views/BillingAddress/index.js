import BillingAddress from './BillingAddress';
import { connect } from 'react-redux';
import { INFO_BILLING_ADDRESS, LOGIN_TYPE } from 'actions/actionTypes';
import { login } from 'actions/user';
import { push } from 'connected-react-router';
import {
    createLoadingSelector,
} from 'utils/selector.helper'
import { listBillingAddressPost, getInfoBillingAddress, contactInformationPut, listBillingAddressGet } from 'actions/billingAddress';
import { getPurchaseSummary, getAttributeName } from 'actions/cart';
import { checkCustomer } from 'actions/category';

//Set selector loading 
const loadingSelector = createLoadingSelector([INFO_BILLING_ADDRESS]);
const mapStateToProps = state => ({
    ...state.user.user,
    isLoading: loadingSelector(state),
    infoBilling: state.billingAddress.infoBilling.data,
    purchaseSummary: state.cart.purchaseSummary.data,
    attributeName: state.cart.attributeName.attributeName,
    billingAddressList: state.billingAddress.billingAddress.data,
    error: state.billingAddress.billingAddress.err,
    dataPost: state.billingAddress.billingAddress.dataPost,
    dataPut: state.billingAddress.billingAddress.dataPutContactInfo,
    loading: state.billingAddress.billingAddress.loading,
    dataCustomer: state.category.customerCode,
});

const mapDispatchToProps = dispatch => ({
    login: (data) => dispatch(login(data)),
    push: (url, state) => dispatch(push(url, state)),
    infoBillingAddress: (userId, token) => dispatch(getInfoBillingAddress(userId, token)),
    fetchGetBillingList: (userId, token) => dispatch(listBillingAddressGet(userId, token)),
    fetchPostBillingList: (data, token) => dispatch(listBillingAddressPost(data, token)),
    fetchPutContactInformation: (data, token) => dispatch(contactInformationPut(data, token)),
    fetchPurchaseSummary: (cartId, token) => dispatch(getPurchaseSummary(cartId, token)),
    fetchAttributeName: (name) => dispatch(getAttributeName(name)),
    checkCustomer: (token) => dispatch(checkCustomer(token)),
});

export default connect(mapStateToProps, mapDispatchToProps)(BillingAddress);