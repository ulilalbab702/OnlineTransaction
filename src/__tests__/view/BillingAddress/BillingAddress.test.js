import React from "react";
import toJson from "enzyme-to-json";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import BillingAddress from "views/BillingAddress/BillingAddress";
import { dataBillingAddressDummy } from "../../../dummy/BillingAddress.dummy";

Enzyme.configure({ adapter: new Adapter() });
describe("Change Address Component", () => {
    const wrapper = shallow(
        <BillingAddress
            data={dataBillingAddressDummy}
        />
    );
    describe("Change Address Component Component", () => {
        it("Should match to snapshot", () => {
            expect(toJson(wrapper)).toMatchSnapshot();
        });
    });
});
