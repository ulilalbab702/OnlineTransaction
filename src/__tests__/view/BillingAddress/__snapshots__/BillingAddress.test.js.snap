// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Change Address Component Change Address Component Component Should match to snapshot 1`] = `
<div>
  <Row
    style={
      Object {
        "marginBottom": "5rem",
        "marginRight": "0",
      }
    }
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      className="contactInformation"
      lg="7"
      md="12"
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
      xl="7"
    >
      <h3
        className="styleCIBold"
      >
        Contact Information
      </h3>
      <Col
        className="formAddress"
        lg="10"
        md="10"
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Form
          tag="form"
        >
          <FormGroup
            className="mb-3"
            tag="div"
          >
            <Label
              for="phoneNumber"
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              Phone Number
            </Label>
            <Input
              className="styleIsiForm"
              defaultValue="081218310111"
              id="phoneNumber"
              maxLength="100"
              name="contactNumber"
              onChange={[Function]}
              onKeyPress={[Function]}
              type="tel"
              value=""
            />
          </FormGroup>
          <FormGroup
            className="mb-1"
            tag="div"
          >
            <Label
              for="email"
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              Email
            </Label>
            <Input
              className="styleIsiForm"
              defaultValue="yohanesKurniawan@mitralestari.com"
              id="contactEmail"
              maxLength="100"
              name="email"
              onChange={[Function]}
              type="email"
              value=""
            />
          </FormGroup>
          <Form
            inline={true}
            style={
              Object {
                "marginBottom": 15,
                "marginTop": 10,
              }
            }
            tag="form"
          >
            <FormGroup
              check={true}
              tag="div"
            >
              <Label
                check={true}
                tag="label"
                widths={
                  Array [
                    "xs",
                    "sm",
                    "md",
                    "lg",
                    "xl",
                  ]
                }
              >
                <Input
                  defaultChecked="checked"
                  type="checkbox"
                />
                 
                Get promo information and updates
              </Label>
            </FormGroup>
          </Form>
          <FormGroup
            className="mb-3"
            tag="div"
          >
            <Label
              for="selectRole"
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              Role
            </Label>
            <Input
              className="styleIsiForm"
              id="selectRole"
              name="position"
              onChange={[Function]}
              type="select"
              value=""
            >
              <option>
                Owner
              </option>
              <option>
                CEO
              </option>
            </Input>
          </FormGroup>
          <hr
            style={
              Object {
                "marginTop": "2.5rem",
                "width": "100%",
              }
            }
          />
          <h3
            className="styleBABold"
          >
            Billing Address
          </h3>
          <FormGroup
            className="mb-3"
            tag="div"
          >
            <Label
              for="fullName"
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              Full Name
            </Label>
            <Input
              className="styleIsiForm"
              defaultValue="Yohanes Kurniawan"
              id="fullName"
              maxLength="100"
              name="fullName"
              onChange={[Function]}
              type="text"
              value=""
            />
          </FormGroup>
          <FormGroup
            className="mb-3"
            tag="div"
          >
            <Label
              for="labelAlamat"
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              Label Alamat
            </Label>
            <Input
              className="styleIsiForm"
              defaultValue="Kantor"
              id="kantor"
              maxLength="100"
              name="addressLabel"
              onChange={[Function]}
              type="text"
              value=""
            />
          </FormGroup>
          <Row
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <Col
              md={9}
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <FormGroup
                className="mb-3"
                tag="div"
              >
                <Label
                  for="kotaKecamatan"
                  tag="label"
                  widths={
                    Array [
                      "xs",
                      "sm",
                      "md",
                      "lg",
                      "xl",
                    ]
                  }
                >
                  Kota atau Kecamatan
                </Label>
                <Input
                  className="styleIsiForm"
                  defaultValue="Perwira, Bekasi Utara, Kota Bekasi"
                  id="kota"
                  maxLength="100"
                  name="city"
                  onChange={[Function]}
                  type="text"
                  value=""
                />
              </FormGroup>
            </Col>
            <Col
              md={3}
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <FormGroup
                className="mb-3"
                tag="div"
              >
                <Label
                  for="kodePos"
                  tag="label"
                  widths={
                    Array [
                      "xs",
                      "sm",
                      "md",
                      "lg",
                      "xl",
                    ]
                  }
                >
                  Kode Pos
                </Label>
                <Input
                  className="styleIsiForm"
                  defaultValue="171222"
                  id="kodePos"
                  maxLength="100"
                  name="postalCode"
                  onChange={[Function]}
                  onKeyPress={[Function]}
                  type="text"
                  value=""
                />
              </FormGroup>
            </Col>
          </Row>
          <FormGroup
            className="mb-3"
            tag="div"
          >
            <Label
              for="alamatDetail"
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              Alamat Detail
            </Label>
            <textarea
              class="styleIsiAlamat"
              defaultValue="Jl Raya kaliabang Tengah, Ruko San Jaya,  Blok A5 No 9, Sebelah Indomaret"
              maxLength="500"
              name="address"
              onChange={[Function]}
              value=""
            />
          </FormGroup>
          <Col
            className="mb-2 p-0"
            lg="12"
            md="12"
            sm="12"
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
            xl="12"
          >
            <Label
              for="alamatDetail"
              tag="label"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              Tandai Lokasi (Opsional)
            </Label>
            <Row
              className="ml-0 boxLokasi"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <div
                className="p-0 iconLokasi"
                lg="1"
                md="2"
                sm="2"
                xl="2"
              >
                <img
                  src="iconLoc.png"
                  style={
                    Object {
                      "height": "51px",
                      "width": "51px",
                    }
                  }
                />
              </div>
              <div
                className="p-0 teksLokasi"
                lg="7"
                md="6"
                sm="6"
                xl="6"
              >
                <div
                  style={
                    Object {
                      "width": "100%",
                    }
                  }
                >
                  Jl. Raya Bekasi No.KM.22, RT.7/RW.1, Cakung Bar., Kec. Cakung, Kota Jakarta Timur, Daerah Khusus Ibukota Jakarta 13910
                </div>
              </div>
              <div
                className="p-0"
                lg="4"
                md="4"
                sm="4"
                xl="4"
              >
                <div
                  className="btnChangePinpoint"
                  onClick={[Function]}
                >
                  <p
                    className="titleBtn"
                  >
                    Change Pinpoint
                  </p>
                </div>
                <ForwardRef(Modal)
                  className="modal-login"
                  onClose={[Function]}
                  open={false}
                >
                  <WithStyles(ForwardRef(DialogContent))
                    className="container-modal-login"
                  >
                    <div
                      className="login-modal-body"
                    >
                      <div
                        className="App"
                      >
                        <LoadScript
                          googleMapsApiKey="AIzaSyBfxiEXIWGI0qyaQ9QMKIjSyYdZ_71e70A"
                          id="script-loader"
                          version="weekly"
                        >
                          <GoogleMap
                            center={
                              Object {
                                "lat": -6.183758441950028,
                                "lng": 106.9311030934085,
                              }
                            }
                            mapContainerStyle={
                              Object {
                                "height": "400px",
                                "width": "400px",
                              }
                            }
                            zoom={10}
                          >
                            <Marker
                              position={
                                Object {
                                  "lat": -6.183758441950028,
                                  "lng": 106.9311030934085,
                                }
                              }
                            />
                          </GoogleMap>
                        </LoadScript>
                      </div>
                    </div>
                  </WithStyles(ForwardRef(DialogContent))>
                </ForwardRef(Modal)>
              </div>
            </Row>
          </Col>
          <hr
            style={
              Object {
                "marginTop": "1.5rem",
                "width": "100%",
              }
            }
          />
          <Col
            className="pr-0"
            lg="12"
            tag="div"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <div
              className="btnContinue"
              onClick={[Function]}
            >
              <p
                className="titleBtn"
                onClick={[Function]}
                style={
                  Object {
                    "color": "#000000",
                  }
                }
              >
                Continue to Delivery
              </p>
            </div>
          </Col>
        </Form>
      </Col>
    </Col>
    <CardInputPromo />
  </Row>
</div>
`;
